program/src/instruction.rs

```diff
@@ -537,41 +492,36 @@ accounts_struct! {
             is_writable: true,
         },
 
-        pub reserve {
-            is_signer: false,
-            // Is writable due to withdraw to reserve (vote_instruction::withdraw)
-            is_writable: true,
-        },
-        // Used to get the rewards out of the validator vote account.
-        pub rewards_withdraw_authority {
-            is_signer: false,
-            is_writable: false,
-        },
+        // Needed for minting rewards.
+        const spl_token_program = spl_token::id(),
```

Comment:
Also here; we can’t change the meaning of the existing instruction, we should add a new instruction instead.